src/main/resources/META-INF/plugin.xml

```diff
@@ -169,6 +169,9 @@
 
         <!-- Structure view -->
         <lang.psiStructureViewFactory language="Robotframework" implementationClass="amailp.intellij.robot.extensions.RobotPsiStructureViewFactory"/>
+
+        <!-- Robot keyword reference -->
+        <psi.referenceContributor implementation="amailp.intellij.robot.psi.reference.KeywordReferenceContributor"/>
```

Comment:
This is needed to make the Python API usage pluggable. There is another reference contributor registered in src/main/resources/META-INF/robot-python.xml. If I leave `getReference` defined in the class then the python reference results will be overridden.